packages/ui/src/components/typeahead/Typeahead.tsx

```diff
@@ -22,36 +32,58 @@ export const Typeahead: FunctionComponent<TypeaheadProps> = ({
   onChange,
   onCleanInput,
   'data-testid': dataTestId,
+  onCreate,
+  onCreatePrefix,
 }) => {
+  const createNewValue = 'create-new';
+  const createNewWithNameValue = 'create-new-with-name';
```

Comment:
since these are static values, could be possible to move them outside of the function so they are not created on every render?